buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -89,7 +89,7 @@ object Config {
 
         // https://developer.android.com/studio/releases/build-tools
         const val BUILD_TOOLS = "31.0.0"
-        const val MIN_SDK = 23 // TODO: TEMPLATE - Replace with appropriate project minSdkVersion
+        const val MIN_SDK = 24 // TODO: TEMPLATE - Replace with appropriate project minSdkVersion
```

Comment:
@MMurrayAtBR While running `lintInternalDebug` locally, it was mentioned that there was a `drawable-v24` resource . That resource should actually be deleted as it uses an incorrect app icon for v24+ devices. The existing resource in drawable is the preferred drawable.

- [ ] Delete `ic_launcher_foreground.xml` from `drawable-v24`